### DISCUSSION TITLE
subscription has ended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40824] [Copilot Conversations]
### Select Topic Area

Bug

### Body

i used to use my copilot for 10 dollar but then i noticed that i can use it for free because i am a student. but when i change my subscription type to student , copilet stoped working.
![copilot1](https://user-images.githubusercontent.com/118359311/205636808-044b2b22-31f7-4995-9e05-f16a019f4fa3.png)
![copilot2](https://user-images.githubusercontent.com/118359311/205636822-7dea1367-ee57-4146-8f77-99bf4b9ff734.png)
as you can see evereything seems ok at my github account. please help me for that 

![copilot3](https://user-images.githubusercontent.com/118359311/205637446-437e8381-30a6-4990-988d-74d156bae783.png)

**Comment by Akash1134** [comment]
Hi @yasir-ozcan The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
